.github/file-filters.yml

```diff
@@ -0,0 +1,4 @@
+# This is used by the action https://github.com/dorny/paths-filter
+
+high_risk_code: &high_risk_code
+  - ".github/file-filters.yml"
```

Comment:
this is for testing and will be removed before merging
need to align on set of potentially higher-risk files before merging as well